src/languageSupport/languages/flux/lsp/monaco.flux.lsp.ts

```diff
@@ -85,6 +90,14 @@ export function initLspWorker() {
   messageWriter = new BrowserMessageWriter(worker)
   const connection = createMessageConnection(messageReader, messageWriter)
   const languageClient = createLanguageClient(connection)
+  languageClient.onTelemetry((data: unknown) => {
+    storeSingleton.dispatch(
```

Comment:
Note: not going to merge this.
Don't plan to use this pattern of a direct store dispatch. Only for fast PoC.